storage/rocksdb_storage.cpp

```diff
@@ -158,6 +171,10 @@ rocksdb_storage::update(const bzn::uuid_t& uuid, const bzn::key_t& key, const bz
         this->update_metadata_size(uuid, NAMESPACE_KEY, SIZE_KEY, ns_prev_size - prev_size + value.size() + key.size());
         this->update_metadata_size(uuid, SIZE_KEY, key, value.size() + key.size());
 
+#ifdef __APPLE__
```

Comment:
perhaps move the flush to the update_metadata_size instead?

Reply:
That's a good idea.

Reply:
Let's create a backlog item to properly understand what is going on here.

Reply:
After discussions with Ed, I've decided the leave the flushes where they are and create a backlog item to find the real problem.